configs/bigcommerce.json

```diff
@@ -1,37 +1,625 @@
+	{"url":"https://support.bigcommerce.com/articles/Public/How-can-I-restrict-wholesale-products-to-wholesale-customers","selectors_key": "docs"},
+	{"url":"https://support.bigcommerce.com/articles/Public/How-do-I-create-and-submit-a-Shoppingcom-MySimon-product-feed","selectors_key": "docs"},
+	{"url":"https://support.bigcommerce.com/articles/Public/How-do-I-Export-Google-AdWords-for-all-of-my-Products-to-a-CSV","selectors_key": "docs"},
+	{"url":"https://support.bigcommerce.com/articles/Public/Adding-the-Facebook-Like-Button-to-Product-and-Blog-Post-Pages","selectors_key": "docs"},
+	{"url":"https://support.bigcommerce.com/articles/Public/Troubleshooting-Missing-or-Inaccurate-Real-Time-Shipping-Quotes","selectors_key": "docs"},
+	{"url":"https://support.bigcommerce.com/articles/Public/Applying-ShipperHQ-Attributes-to-Products-via-CSV-Import-Export","selectors_key": "docs"},
+	{"url":"https://support.bigcommerce.com/articles/Public/Removing-Your-Logo-from-Your-Store-s-Invoices-and-Packing-Slips","selectors_key": "docs"},
+	{"url":"https://support.bigcommerce.com/articles/Public/Why-does-the-carousel-preview-look-different-than-my-storefront","selectors_key": "docs"},
+	{"url":"https://support.bigcommerce.com/articles/Public/How-do-I-create-and-submit-a-Bizrate-Shopzilla-Beso-product-feed","selectors_key": "docs"},
+	{"url":"https://support.bigcommerce.com/articles/Public/Adding-a-Shipping-Zone-for-the-Rest-of-the-World-Everywhere-Else","selectors_key": "docs"},
+	{"url":"https://support.bigcommerce.com/articles/Public/ShipperHQ-Adding-the-City-Field-to-the-Cart-Page-Blueprint-Themes","selectors_key": "docs"},
+	{"url":"https://support.bigcommerce.com/articles/Public/How-can-I-hide-a-product-but-still-make-it-available-for-purchasing","selectors_key": "docs"},
+	{"url":"https://support.bigcommerce.com/articles/Public/Limiting-States-Provinces-on-Account-Creation-View-Cart-Checkout-Pages","selectors_key": "docs"},
+	{"url":"https://support.bigcommerce.com/articles/Public/How-do-my-customers-reset-their-password-for-their-account-on-my-store","selectors_key": "docs"},
+	{"url":"https://support.bigcommerce.com/articles/Public/How-do-I-change-the-Unable-to-Ship-to-this-Location-message-on-checkout","selectors_key": "docs"},
+	{"url":"https://support.bigcommerce.com/articles/Public/How-do-I-auto-return-my-customers-to-my-store-after-they-checkout-with-PayPal","selectors_key": "docs"},
+	{"url":"https://support.bigcommerce.com/articles/Public/How-do-I-control-the-display-order-of-the-products-on-my-category-and-brand-pages","selectors_key": "docs"},
+	{"url":"https://support.bigcommerce.com/articles/Public/How-Bigcommerce-Calculates-Trailing-12-Month-Sales-Orders-to-Determine-Your-Pricing","selectors_key": "docs"},
+	{"url":"https://support.bigcommerce.com/articles/Public/How-do-I-access-and-install-the-new-single-click-apps-within-my-Bigcommerce-store-control-panel","selectors_key": "docs"},
+	{"url":"https://support.bigcommerce.com/articles/Public
```

Comment:
can't you use a linter? like `jq`? 
It allows to have better diff when modifying the file